src/main/webapp/WEB-INF/jsp/viewAllSubjectSDVtmp.jsp

```diff
@@ -554,6 +554,15 @@
             getItems();
         }).change();
 
+        $('#clear-filter').click(function(){
+                jQuery('#sdv-items').dataTable().fnSortNeutral();   
```

Comment:
why are you sorting the items here?

Reply:
This is what done previously in clearFilter() function (line 421) to remove the filters on each column, So I used the same and added toggle to defaultRadioButton

Reply:
why not update the existing clearFilter() function?

Reply:
data was not able to accessed from clearFunction(), So I created a new function inside it

Reply:
Altered the code to use clearFilter() method in all places